examples/plugin/Hello_world/Hello_world.cc

```diff
@@ -0,0 +1,40 @@
+#include <ignition/gazebo/Model.hh>
+#include <ignition/gazebo/System.hh>
+#include <ignition/gazebo/Util.hh>
+
+
+namespace ignition;
+namespace gazebo;
+namespace systems;
+
+// Inherit from System and 2 extra interfaces
+// ISystemConfigure and ISystemPostUpdate
+
+class ignition::gazebo::systems::Hello_world
+
+{   
+};
+/// \ brief Hello_world data class
+Hello_world::Hello_world() : System(), dataPtr(std::make_unique<Hello_world>())
+
+//////////////////////////////////////////////////////////////
+
+Hello_world::~Hello_world() = default;
+
+// Implement Configure callback, provided by ISystemConfigure
+// and called once at startup.
+void MyPlugin::Configure(const Entity &_entity,
+    const std::shared_ptr<const sdf::Element> &_sdf,
+    EntityComponentManager &_ecm,
+    EventManager &/*_eventMgr*/)
+
+{
+	printf("Hello World!\n");
```

Comment:
Use 2-space indentation

```suggestion
  printf("Hello World!\n");
```